script/BetterScript.s.sol

```diff
@@ -2,26 +2,36 @@
 pragma solidity ^0.8.23;
 
 import {Script, console2 as console} from "forge-std/Script.sol";
+import {Sphinx, Network} from "@sphinx-labs/plugins/SphinxPlugin.sol";
 
 interface ICREATE3Factory {
     function deploy(bytes32 salt, bytes memory creationCode) external payable returns (address deployed);
 
     function getDeployed(address deployer, bytes32 salt) external view returns (address deployed);
 }
 
-contract BetterScript is Script {
+contract BetterScript is Script, Sphinx {
     ICREATE3Factory internal factory = ICREATE3Factory(0x2Dfcc7415D89af828cbef005F0d072D8b3F23183);
 
-    modifier broadcast(uint256 key) {
-        vm.startBroadcast(key);
-        _;
-        vm.stopBroadcast();
-    }
-
-    modifier broadcastAddr(address who) {
-        vm.startBroadcast(who);
-        _;
-        vm.stopBroadcast();
```

Comment:
Removed the broadcast logic because it's not necessary anymore. Sphinx impersonates your deployer (i.e. your Gnosis Safe) in the `sphinx` modifier, which was added to your script's `run()` function.